assets/src/containers/ResourcesAccessed.js

```diff
@@ -51,7 +51,7 @@ function ResourcesAccessed (props) {
   const [curWeek, setCurWeek] = useState(0) // Should be updated from info
   const [weekRange, setWeekRange] = useState([]) // Should be depend on curWeek
   const [gradeRangeFilter, setGradeRangeFilter] = useState('') // Should be fetched from default
-  const [resourceFilter, setResourceFilter] = useState([])
+  const [resourceFilter, setResourceFilter] = useState(resourceTypes)
```

Comment:
Because of this, I see resource request call made as below on loading of the Resources view
`resource_access_within_week?week_num_start=10&week…grade=All&resource_type=Files,Videos,Files,Videos`
You can look at this from Chrome Developer tools: Network tab

Reply:
I think the duplicates are set in the `useEffect` from line 142 - 158. You don't need that anymore if you use `useState(resourceTypes)`

Reply:
`const [resourceFilter, setResourceFilter] = useState(resourceTypes)`
the `resourceTypes` values can be `[], ['Files', 'Videos'], ['Files'],['Videos']`
i think in case of empty value i feel you can default it to `['Files']`